doc/Language/objects.pod6

```diff
@@ -4,8 +4,8 @@
 
 =SUBTITLE Object Orientation in Perl 6
 
-Perl 6 is an object oriented language at its core, even though it allows you
-to write programs in other programming styles.
+Perl 6 is an object oriented language at its core, although you can write
```

Comment:
If you want to keep this sentence as is, it should be “Although Perl 6 is an object oriented language at its core, you can write programs in other programming styles”.

It would be much better if this had two sentences about Perl 6 being an object oriented language, and then another after that says that you are allowed to write code in other programming styles as well.